clippy_lints/src/utils/diagnostics.rs

```diff
@@ -163,6 +163,7 @@ pub fn span_lint_hir_and_then(
 ///     |
 ///     = note: `-D fold-any` implied by `-D warnings`
 /// ```
+#[allow(clippy::collapsible_span_lint_calls)]
```

Comment:
Adding this does not work. The terminal emits an unknown lint error. I can't figure out why.

Reply:
`update_lints` is broken for internal lints. I'm working on a fix